scanomatic/models/analysis_model.py

```diff
@@ -23,12 +23,24 @@ class IMAGE_ROTATIONS(Enum):
     Portrait = 1
     Unknown = 2
 
+    @classmethod
+    def from_name(cls, raw_string: str) -> "IMAGE_ROTATIONS":
```

Comment:
why not just `IMAGE_ROTATIONS['Unknown']`?

Reply:
or are some recorded in lower-case? and if this function really is needed then it needs to raise value error if no member found

Reply:
but I really doubt we need it since we could load all serialized states before with jsonizer.